main/src/library/Files.flix

```diff
@@ -894,14 +775,12 @@ mod Files {
     ///
     pub def move(src: {src = String} , dst: String): Result[IOError, Unit] \ IO = region rc {
         IOError.tryCatch(_ -> {
-            import java.io.File.toPath(): Path \ IO;
-            import static java.nio.file.Files.move(Path, Path, Array[CopyOption, rc]): Path \ IO;
-
             let srcFile = new File(src#src);
-            let srcPath = toPath(srcFile);
+            let srcPath = srcFile.toPath();
             let dstFile = new File(dst);
-            let dstPath = toPath(dstFile);
-            discard move(srcPath, dstPath, Array#{} @ rc)
+            let dstPath = dstFile.toPath();
+            let empty: Array[CopyOption, _] = Array#{} @ rc;
```

Comment:
There is a bunch of these empty arrays that need to be annotated, but I guess that's fine